app/src/main/java/woowacourse/shopping/data/goods/repository/GoodsRemoteDataSourceImpl.kt

```
@@ -0,0 +1,180 @@
+        okHttpClient.newCall(request).enqueue(
+            object : Callback {
+                override fun onResponse(
+                    call: Call,
+                    response: Response,
+                ) {
+                    response.body?.string()?.let { jsonString ->
+                        try {
+                            val jsonObject = gson.fromJson(jsonString, JsonObject::class.java)
+                            val size = jsonObject.get("size")?.asInt ?: 0
+                            onComplete(size)
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)